### PR TITLE
[TF] Add TF installation instructions

### PR DESCRIPTION
**Pull request by terrytangyuan**
*Description of changes:*

By submitting this pull request, I confirm that you can use, modify,
copy, and redistribute this contribution, under the terms of your
choice.

### CODE REVIEW

**Comment by mli** [comment]
Job d2l-en/PR-1064/1 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1064/

**Comment by terrytangyuan** [comment]
@mli @astonzhang It doesn't seem like tab toggle for sections like `GPU Support` works. Any suggestions? Feel free to modify this branch directly. Currently the section `GPU Support` are appears separately for MXNet and PyTorch where I expect to see this section only pops up for MXNet and PyTorch.

**Comment by astonzhang** [comment]
@mli can you take a look at the tab issue as mentioned by yuan?

File: chapter_installation/index.md [hunk 1]
```diff
@@ -112,10 +123,11 @@ jupyter notebook
 
 At this point, you can open http://localhost:8888 (it usually opens automatically) in your Web browser. Then we can run the code for each section of the book.
 Please always execute `conda activate d2l` to activate the runtime environment
-before running the code of the book or updating MXNet or the `d2l` package.
+before running the code of the book or updating the framework or the `d2l` package.
 To exit the environment, run `conda deactivate`.
 
 
+:begin_tab:`mxnet,pytorch`
```
Comment (mli) [inline chapter_installation/index.md:130]: it's not supported yet to have a section in a begin_tab. move this line below `## GPU Support` should fix the issue

**Comment by mli** [comment]
Job d2l-en/PR-1064/3 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1064/

**Comment by terrytangyuan** [comment]
@mli Thanks. Fixed.

**Pull request merged**